contracts/child/ChildERC20.sol

```diff
@@ -74,7 +46,7 @@ contract ChildERC20 is ChildToken, ERC20, LibTokenTransferOrder, ERC20Detailed {
    *
    * @param amount tokens
    */
-  function withdraw(uint256 amount) public {
+  function withdraw(uint256 amount) public payable {
```

Comment:
Commented yesterday as well, we shouldn't make these payable to resolve a compilation error. Need to find another way to make it payable in childMatic.

Reply:
yes looking for it.